app/assets/stylesheets/application.css

```diff
@@ -123,6 +123,11 @@ form input[type="text"]:focus {
   box-shadow: 0 2px 2px -2px rgba(0,0,0,.2);
 }
 
+.error-message {
+  color: red;
```

Comment:
I like that you added this color here so that our error message stands out a bit more